docs/v3/group-chat.md

```diff
@@ -0,0 +1,165 @@
+# Group Chat
+
+## Summary
+
+* XMTP V3 is introducing Group Chat based on the MLS [specification](https://www.rfc-editor.org/rfc/rfc9420.html).
+* A group can be anywhere from 1 person to 20,000.
+* Groups can have administrators with special permissions. Those permissions include:
+  * Add or remove group members.
+  * Mute group members.
```

Comment:
I don't think there is any admin ability to mute members for everyone. Members can be "muted" on a individual bases but not for everyone else.

Reply:
That seems like an odd choice for large groups.  To clarify: a user can mute a user from their own feed, but administrators cannot mute group members?  You just have to straight to removal?

Reply:
Correct. @neekolas or @cameronvoell have we thought about this feature for groups? Technically speaking I'm not sure how we would even do that.

Reply:
We discussed it one of the meetings but that doesn't mean we implemented it.  It's just super common when you want a one-to-many discussion or to keep large public chats civil.

Reply:
We had a conversation about this with the MLS team today. Seems like this is definitely something we will eventually want but not something we will currently support. Lets remove it from the docs for now.

Reply:
No worries, I removed it yesterday so it's gone from the current build.